libs/wire-api/src/Wire/API/Routes/Version.hs

```diff
@@ -69,7 +69,8 @@ instance ToSchema Version where
       [ element 0 V0,
         element 1 V1,
         element 2 V2,
-        element 3 V3
+        element 3 V3,
+        element 4 V4
```

Comment:
Recently we've had that discussion about using versions in endpoint paths and how we missed a bug because we were not testing what versions get converted to when made a path component. Do you want to add a test for it in `Test.Wire.API.Routes.Version`?

Reply:
What test do you mean?

Reply:
In `Test.Wire.API.Routes.Version` I'd extend the test like so:

```haskell
testToPathComponent :: Assertion
testToPathComponent = do
  "v0" @=? toPathComponent V0
  "v1" @=? toPathComponent V1
  "v2" @=? toPathComponent V2
  "v3" @=? toPathComponent V3
  "v4" @=? toPathComponent V4
```